### ISSUE TITLE
Stop working with https://web.telegram.org/k

### ISSUE THREAD

**Issue by ytzhhws** [Neet-Nestor/Telegram-Media-Downloader#28]
Hi, thanks for your code, it worked perfectly for me during the past week. Unfortunately since yesterday it stopped working when I use https://web.telegram.org/k. The download icons disappeared. I reinstalled both Tampermonkey and your downloader but still not working. 

Interestingly, when I switch to https://web.telegram.org/a, the download icons appeared and I could download the videos, but very slow, and I have to wait until the progress to 100% in background then the browser will save the file, like you described. The file name seems to be random as well. However, when it worked well with the telegram K version last week, I could see the download progress in Chrome downloads icon, just like normal downloading, and the file name is the real name from the telegram. 

Another thing is when I use Firefox, the download still does not work with https://web.telegram.org/k, no download icons just the same as in Chrome. However the icons will appear in https://web.telegram.org/a, but when I try to download, the file saved is only 9 bytes for every video. 

In summary, now the downloader only works on Chrome with https://web.telegram.org/a, and seems to be less effective compared with previous time. I guess it is due to some updates from Telegram? Look forward to your update version to fix these bugs. It is a really good program, thank you guys!

**Comment by Neet-Nestor** [comment]
Thanks for reporting. I will check it at a later time this week.

**Comment by covidhhle** [comment]
Hello! 

Not a web dev by any means, but the button is still there, just invisible because Telegram /k/ now uses a span class inside the button to display the icon. I fixed it by adding the equivalent span to the button. 

downloadButton.innerHTML += "<span class=\"tgico button-icon\"></span>";

![2023-09-07 16_28_08-Telegram Media Downloader](https://github.com/Neet-Nestor/Telegram-Media-Downloader/assets/35625712/2687395e-121d-4c71-bee3-73d6e7bee62d)

**Comment by Neet-Nestor** [comment]
@covidhhle Thanks for the investigation and proposing this solution! I'll validate this idea tonight asap and update the script accordingly.

**Comment by ytzhhws** [comment]
Thanks for working on it, look forward to the update!

**Comment by akec01** [comment]
@covidhhle your solution worked for me. The only issue was that I had to change " for ' for it work.

**Comment by Neet-Nestor** [comment]
Fixed with 1be7b2ace9751a72f447c56c446b592b5200dbde.
